libtcmu.c

```diff
@@ -383,18 +383,192 @@ static void tcmu_cdb_debug_info(const struct tcmulib_cmd *cmd)
 		free(buf);
 }
 
-static void *tcmu_cmdproc_thread(void *arg)
+static void _cleanup_mutex_lock(void *arg)
+{
+	pthread_mutex_unlock(arg);
+}
+
+static void *aio_work_queue(void *arg)
```

Comment:
Nit: am I correct that this is really just an IO worker thread to wrap synchronous IO calls? If so, perhaps instead of naming it "aio", can it just be "io"?